playbooks/base-minimal-test/pre.yaml

```diff
@@ -16,7 +16,10 @@
   tasks:
     - name: Run add-build-sshkey role
       include_role:
-        name: add-build-sshkey
+        name: add-build-sshkey-fork
+      vars:
+        zuul_ssh_key_algorithm: "ecdsa"
+        zuul_ssh_key_size: "521"
```

Comment:
so, we'll want to remove this for now. To confirm the defaults still work.

Then at the job level, we can toggle this to ecdsa.